app/en/home/auth-providers/asana/page.mdx

```diff
@@ -139,12 +134,7 @@ ASANA_CLIENT_SECRET="<your client secret>"
 
 #### Edit the Engine configuration
 
-<Callout>
-  To locate the `engine.yaml` file in your OS after installing the Arcade
-  Engine, check the [Engine configuration
-  file](/home/local-deployment/configure/overview#engine-configuration-file)
-  documentation.
-</Callout>
+
 
 Edit the `engine.yaml` file and add an Asana item to the `auth.providers` section:
```

Comment:
@evantahler Why not remove all instructions related to editing `engine.yaml`?

Reply:
I was torn about this - do you think we need to keep this around for our enterprise customers?

Reply:
I think it's almost time for a set of enterprise documentation!